rllib/core/rl_trainer/rl_trainer.py

```diff
@@ -216,14 +217,32 @@ def compile_results(
         Returns:
             A dictionary of results.
         """
+        # TODO (Kourosh): This method assumes that all the modules with in the
```

Comment:
nits:
-> "within"
-> ".. its `self._rl_modules` ..." ?

Reply:
Also, can we fix the inconsistency between `postprocessed_loss` vs `post_processed_gradients`?
Everywhere else in RLlib, we use one word: "postprocess.."